src/api.ts

```diff
@@ -26,3 +26,35 @@ function get<T>(url: string): Promise<T> {
 export const getTodos = () => get<Todo[]>('/todos');
 
 export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+
+export const getSortedTodos = () => {
+  return getTodos().then(todos => todos.sort((a, b) => a.id - b.id));
```

Comment:
Be cautious when initializing state with props or derived state. If the todos' order is meant to be maintained in the state, any changes to the todos in the state won't be reflected in the sorted todos unless the component is re-rendered with new props. Consider if this is the desired behavior or if sorting should be handled within the component's state logic.